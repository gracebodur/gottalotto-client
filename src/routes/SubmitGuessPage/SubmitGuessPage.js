import React, { Component } from 'react'
import SubmitGuessForm from '../../components/SubmitGuessForm/SubmitGuessForm';

class SubmitGuessPage extends Component {
	render() {
		return (
			<div>
				<SubmitGuessForm />
			</div>
		)
	}
}

export default SubmitGuessPage