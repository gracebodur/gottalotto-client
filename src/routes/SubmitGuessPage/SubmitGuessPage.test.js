import React from "react";
import toJson from "enzyme-to-json";
import SubmitGuessPage from "./SubmitGuessPage";

describe(`SubmitGuessPage component`, () => {
	const props = {
		id: "a",
		name: "test-class-name"
	};

	it("renders a SubmitGuessPage by default", () => {
		const wrapper = <SubmitGuessPage />;
		expect(toJson(wrapper)).toMatchSnapshot();
	});

	it("renders the SubmitGuessPage given props", () => {
		const wrapper = <SubmitGuessPage {...props} />;
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
