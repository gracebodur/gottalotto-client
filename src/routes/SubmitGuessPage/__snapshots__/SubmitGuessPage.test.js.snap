// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubmitGuessPage component renders a SubmitGuessPage by default 1`] = `undefined`;

exports[`SubmitGuessPage component renders the SubmitGuessPage given props 1`] = `undefined`;
